dirty_cat/test/test_datetime_encoder.py

```diff
@@ -0,0 +1,183 @@
+import pandas as pd
+from dirty_cat.datetime_encoder import DatetimeEncoder
+import numpy as np
+
+
+def get_date_array():
+    return np.array([pd.to_datetime(["2020-01-01", "2020-01-02", "2020-01-03"]),
+                  pd.to_datetime(["2021-02-03", "2020-02-04", "2021-02-05"]),
+                  pd.to_datetime(["2022-01-01", "2020-12-25", "2022-01-03"]),
+                  pd.to_datetime(["2023-02-03", "2020-02-04", "2023-02-05"])])
+
+def get_datetime_array():
+    return np.array([pd.to_datetime(["2020-01-01 10:12:01", "2020-01-02 10:23:00", "2020-01-03 10:00:00"]),
+                  pd.to_datetime(["2021-02-03 12:45:23", "2020-02-04 22:12:00", "2021-02-05 12:00:00"]),
+                  pd.to_datetime(["2022-01-01 23:23:43", "2020-12-25 11:12:00", "2022-01-03 11:00:00"]),
+                  pd.to_datetime(["2023-02-03 11:12:12", "2020-02-04 08:32:00", "2023-02-05 23:00:00"])])
```

Comment:
PEP8 style guide says that there should be 2 empty lines between first-level functions (comment applies in quite a few places)